src/ui-layout.js

```diff
@@ -67,7 +75,6 @@ angular.module('ui.layout', [])
 
     var debounceEvent;
     function draw() {
-      var position = ctrl.sizeProperties.flowProperty;
```

Comment:
I removed the dependency of the code on the flow property variable. It only remains when setting the css. For one, this is a refactoring step that makes the code more readable in my opinion. But more importantly it is also more robust, because the position keeps being valid even if the flow property changes (in case the layout is switched to RTL for example). Because before the position was stored in `container.left` for example, which turns into `container.right` when switching to RTL, a problem that I wanted to avoid.